images/argocd-repo-server/tests/main.tf

```diff
@@ -0,0 +1,46 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "run" {
+  digest = var.digest
+  script = "docker run --rm $${IMAGE_NAME} --help"
+}
+
+resource "random_string" "random" {
+  length  = 6
+  upper   = false
+  special = false
+}
+
+resource "helm_release" "argocd-repo-server" {
```

Comment:
I want to confirm this waits for a meaningful installation signal?

Reply:
👍 the meaningfulness of the signal will depend on the chart, but at a bare minimum the provider defaults to waiting for all resources to be `Ready`

Reply:
There's a `wait` parameter that's true by default, which waits for the jobs to become ready.

Reply:
There's also `wait_for_jobs` (default false) that

> will wait until all Jobs have been completed before marking the release as successful. It will wait for as long as timeout. Defaults to false.